File: crop-samarica-admin/src/app/models/Developers.ts
export interface Developers {
  id: string;
  name: string;
  profile: string;
  email: string;
  roles: DeveloperRoles[];
}

export enum DeveloperRoles {
  ANDRIOD_DEVELOPER = 'ANDROID DEVELOPER',
  WEB_DEVELOPER = 'WEB DEVELOPER',
  UI_UX_DESIGNER = 'UI/UX DESIGNER',
  RESEARCHER = 'RESEARCHER',
  DATA_ANALYST = 'DATA ANALYST',
}

File: CropSamarica/app/src/main/java/com/jmballangca/cropsamarica/domain/models/Developers.kt
package com.jmballangca.cropsamarica.domain.models





data class Developers(
    val id : String = "",
    val name : String = "",
    val email : String = "",
    val profile : String = "",
    val roles: List<String> = emptyList(),
)



enum class DeveloperRoles(val displayName: String) {
    ANDROID_DEVELOPER("Android Developer"),
    WEB_DEVELOPER("Web Developer"),
    UI_UX_DESIGNER("UI/UX Designer"),
    RESEARCHER("Researcher"),
    DATA_ANALYST("Data Analyst")
}
